File: classificarGamer.java
public class classificarGamer {

    public static String classifyPlayer(int victories, int defeats) {
        int balance = victories - defeats;
        String level= "";

        if (balance <= 10) {
            level = "Ferro";
        } else if (balance > 10 && balance <= 20) {
            level = "Bronze";
        } else if (balance > 20 && balance <= 50) {
            level = "Prata";
        } else if (balance > 50 && balance <= 80) {
            level = "Ouro";
        } else if (balance > 80 && balance <= 90) {
            level = "Diamante";
        } else if (balance > 90 && balance <= 100) {
            level = "Lendário";
        } else if (balance > 100 && balance <= 200) {
            level = "Imortal";
        } else {
            level = "Antisocial";
        }
        return "O Antiherói tem um saldo de " + balance + " vitórias e está no nível de " + level;
    }

    public static void main(String[] args) {
        int victories = 66;
        int defeats = 21;
        String result = classifyPlayer(victories, defeats);
        System.out.println(result);
    }
}

File: classificarGamer.js
function classificarGamer(vitorias, derrotas) {
  const saldoVitorias = vitorias - derrotas;
  let nivel = "";
  
  if (saldoVitorias < 10) {
    nivel = "Ferro";
  } else if (saldoVitorias <= 20 && saldoVitorias >= 11) {
    nivel = "Bronze";
  } else if (saldoVitorias <= 50 && saldoVitorias >= 21) {
    nivel = "Prata";
  } else if (saldoVitorias <= 80 && saldoVitorias >= 51) {
    nivel = "Ouro";
  } else if (saldoVitorias <= 90 && saldoVitorias >= 81) {
    nivel = "Diamante";
  } else if (saldoVitorias <= 100 && saldoVitorias >= 91) {
    nivel = "Lendário";
  } else if (saldoVitorias <= 120 && saldoVitorias >= 101) {
    nivel = "Imortal";
  } else {
    nivel = "Antisocial";
  }
  
  return `O Antiherói tem um saldo de ${saldoVitorias} vitórias e está no nível de ${nivel}`;
}
  
const vitorias = 50;
const derrotas = 30;
let resultado = classificarGamer(vitorias, derrotas)
console.log(resultado);